aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -207,6 +207,10 @@
       "description": "The Amazon Resource Name (ARN) of the source DB instance or DB cluster if this DB cluster is created as a Read Replica.",
       "type": "string"
     },
+    "RestoreToTime": {
+      "description": "The date and time to restore the DB cluster to. Value must be a time in Universal Coordinated Time (UTC) format. An example: 2015-03-07T23:45:00Z",
```

Comment:
Given this is a new attribute, I believe we should add a regex-based validation here. It would compile in cryptic message, but this description would explain it.

Reply:
Done.